composer.json

```diff
@@ -36,5 +36,25 @@
             "dealerdirect/phpcodesniffer-composer-installer": true
         },
         "lock": false
+    },
+    "scripts": {
+        "phplint": "parallel-lint -j 10 --exclude vendor --exclude bin .",
```

Comment:
`-j <num> Run jobs in parallel (default: 10).` tak to tam ani nemusi byt ;-) Lepsi by bylo mit tam `-j $(nproc)`
https://github.com/JakubOnderka/PHP-Parallel-Lint#options-for-run

Reply:
dik bral som to z inej libky a tam je to, tak som to uplne vyhodil nedaval by som tam tu premennu